packages/stargate/src/signingstargateclient.spec.ts

```diff
@@ -690,6 +690,329 @@ describe("SigningStargateClient", () => {
     });
   });
 
+  describe("signAndBroadcastSync", () => {
+    describe("direct mode", () => {
+      it("works", async () => {
+        pendingWithoutSimapp();
+        const wallet = await DirectSecp256k1HdWallet.fromMnemonic(faucet.mnemonic);
+        const client = await SigningStargateClient.connectWithSigner(
+          simapp.tendermintUrl,
+          wallet,
+          defaultSigningClientOptions,
+        );
+
+        const msg = MsgDelegate.fromPartial({
+          delegatorAddress: faucet.address0,
+          validatorAddress: validator.validatorAddress,
+          amount: coin(1234, "ustake"),
+        });
+        const msgAny: MsgDelegateEncodeObject = {
+          typeUrl: "/cosmos.staking.v1beta1.MsgDelegate",
+          value: msg,
+        };
```

Comment:
Okay, looking at the CI failure we need a different message type here. The additional MsgDelegate makes an unrelated staking query fail. When we change to some simple MsgSend the problem should go away.

Reply:
Done